types/node-static/index.d.ts

```diff
@@ -42,7 +42,7 @@ export class Server {
 	finish: (status: number, headers: Headers, req: http.ServerRequest, res: http.ServerResponse, promise: events.EventEmitter, callback: Callback) => void;
 	servePath: (pathname: string, status: number, headers: Headers, req: http.ServerRequest, res: http.ServerResponse, finish: Finish) => events.EventEmitter;
 	resolve: (pathname: string) => string;
-	serve: (req: http.ServerRequest, res: http.ServerResponse, callback: Callback) => events.EventEmitter;
+	serve: (req: http.ServerRequest, res: http.ServerResponse, callback?: Callback) => events.EventEmitter;
```

Comment:
The return value can be undefined in some cases

Reply:
can you be more specific about how that's the case with optional callback vs before this change?